array/js/array.js

```diff
@@ -0,0 +1,92 @@
+function isArray(object) {
+  return Array.isArray(object);
+}
+
+function isArray2(object) {
+  return object.constructor.name == "Array";
+}
+
+function isArray3(object) {
+  return {}.toString.call(object) == "[object Array]";
+}
+
+function range(start, end, step) {
```

Comment:
please look at requirements for function range(5) should return [0, 1, 2, 3, 4]
(1, 5) ->  [1, 2, 3, 4]